DDCE-758New:Added the required chnages to the messages

Deep547 opened the pull request.

Ufuoma-Daniel commented:
How come there is no checklist on this pull request? The steps will still need to be followed e.g. squashing commits.

Ufuoma-Daniel commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -0,0 +1,38 @@
+# DDCE-758
+
+##Bug fix / New feature (delete as appropriate)
+
```
I think there's been some confusion here. You should fill out the checklist after you raise the pull request and not editing the template directly. If not, everytime anyone raises a pull request it will now have all the things you've filled in here.

Deep547 commented:
# DDCE-758

##Bug fix / New feature (delete as appropriate)

ERS had a detailed design review and on alterations-or-a-variation it was identified that the error message for no selection of radio button needs updating.

Changes were made to 4 messages altogether. Variables ers_alt_activity.csop.err.message & ers_alt_activity.saye.err.message in the file "conf/messages.cy" had their messages as "Cadarnhewch a oes addasiadau wedi eu gwneud neu a oes amrywiad wedi bod yn y cyfalaf cyfranddaliadau", and the message was changed to "Dewiswch ‘Iawn’ os yw addasiadau wedi eu gwneud i nodweddion allweddol y cynllun hwn neu os yw amrywiad wedi bod yn y cyfalaf cyfranddaliadau".

Similarly, the variables ers_alt_activity.csop.err.message & ers_alt_activity.saye.err.message in the file "conf/messages.en" had their values as "Confirm if alterations have been made or there’s been a variation in the share capital" and the message was changed to "Select yes if alterations have been made to key features of this scheme or there has been a variation in the share capital".

PR Link:
https://github.com/hmrc/ers-returns-frontend/compare/master...DDCE-758New

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?: Yes
- Have you done a visual check of the changes?: Yes
- Is there any still commented or unused code? Lingering printlns?: No
- Have things been implemented in a complicated way?: No
- Are the test still over the coverage threshold?: N/A
- Does the compiler throw a lot of warnings?: No
- Have methods and tests been named clearly?: N/A
- Were there any changes in the config? Do changes need to be made in app-config-??? No
- Have you done a manual walkthrough? Yes


## Checklist PR Raiser
 - [✓]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [✓]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [✓]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [✓]  I've squashed my commits - including the JIRA issue number in the commit message
 - [✓]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've verified every effort was made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've verified appropriate tests were included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've verified commits were squashed and rebased - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

DannyPennington97 commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -22,10 +22,18 @@ Please include a summary / description of the change and which issue it fixes.
  - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
  - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
  - [ ]  I've run a dependency check to ensure all dependencies are up to date
+<<<<<<< 9ea59b5b37dd5a5b3263a820dbabef0bea30b305
```
Please remove leftover artifacts from the rebase

DannyPennington97 commented:
Also it looks like the commit is still not signing correctly as merging is currently blocked
